db/app.js

```diff
@@ -5,27 +5,38 @@ const {
   getArticles,
   getCommentsByArticleId,
   addCommentByArticleId,
+  updateArticleById,
 } = require("./controllers/controller.js");
 const app = express();
 const endpoints = require("../endpoints.json");
 
 app.use(express.json());
 
 //middleware
+
+// task 2
```

Comment:
Remember to remove comments, if they are really helping you now then do it at the end of the sprint. If a potential employer is looking at this repo, the task number is going to be meaningless to them and it looks a little unprofessional